utils/forest_utils/tests/macros_test.rs

```diff
@@ -0,0 +1,56 @@
+// Copyright 2019-2023 ChainSafe Systems
+// SPDX-License-Identifier: Apache-2.0, MIT
+
+use std::time::Duration;
+
+use forest_utils::retry;
+use log::info;
+use tokio::time::sleep;
+
+async fn failing_function() -> Result<(), ()> {
+    sleep(Duration::from_nanos(0)).await;
```

Comment:
Why do we need to `sleep(0)`? Isn't it virtually the same as not sleeping at all?

Reply:
this is for lint check actually, I needed await command inside of async fn

Reply:
Makes sense. It's not a huge deal, so I'm fine with this.

Reply:
Alternatively, we could suppress the linter warning in those few methods. It'd be more self-explaining.

Reply:
I suppressed the linter warning for those 3 methods